src/providers/twitch/TwitchEmotes.cpp

```diff
@@ -75,7 +75,7 @@ EmotePtr TwitchEmotes::getOrCreateEmote(const EmoteId &id,
     if (!shared)
     {
         (*cache)[id] = shared = std::make_shared<Emote>(Emote{
-            EmoteName{name},
+            EmoteName{this->cleanUpEmoteCode(EmoteName{name})},
```

Comment:
```suggestion
            EmoteName{TwitchEmotes::cleanUpEmoteCode(EmoteName{name})},
```
Just a small suggestion which does not change any logic but is more conventional, I think. This just makes it more obvious to the reader that `TwitchEmotes::cleanUpEmoteCode` is a static method.

Reply:
Sure, I agree. I just thought that's it's better to use `this` since it's possible to do it in that context - making code a bit shorter.